From 1d5ceafdbc005f3adc58be5af1dc92f068908e16 Mon Sep 17 00:00:00 2001
From: Artem Gindinson <artem.gindinson@intel.com>
Date: Mon, 20 Jun 2022 12:38:04 +0000
Subject: [PATCH]  Wrap CallInst::getNumArgOperands calls for LLVM 14

Rework all remaining call sites to rely on the already-existing wrapper
function.
---
 IGC/Compiler/CISACodeGen/HFpackingOpt.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/IGC/Compiler/CISACodeGen/HFpackingOpt.cpp b/IGC/Compiler/CISACodeGen/HFpackingOpt.cpp
index 69d6151f8bf..e24362d7791 100644
--- a/IGC/Compiler/CISACodeGen/HFpackingOpt.cpp
+++ b/IGC/Compiler/CISACodeGen/HFpackingOpt.cpp
@@ -825,7 +825,7 @@ bool HFpackingOpt::findStoreSequence(std::vector<Instruction*>& path, std::vecto
         uint srciCount = inst->getNumOperands();
         if (CallInst* cinst = dyn_cast<CallInst>(inst))
         {
-            srciCount = cinst->getNumArgOperands();
+            srciCount = IGCLLVM::getNumArgOperands(cinst);
         }
 
         for (uint srci = 0; srci < srciCount; srci++)
